src/rctmon/device_manager.py

```diff
@@ -448,6 +448,18 @@ def _cb_grid(self, oid: int, value: Any) -> None:
             # g_sync.u_l_rms[2]
             elif oid == 0x2545E22D:
                 self.readings.grid.voltage_l3 = ensure_type(value, float)
+            # g_sync.u_ptp_rms[0]
```

Comment:
fyi: These are not really related to the intended change, but I noticed they were defined and requested, but just not populated for prometheus.